backend/Hortogram/Hortogram/Controllers/UserController.cs

```diff
@@ -91,10 +91,44 @@ public IActionResult Put([FromQuery] Guid id, [FromBody] string firstName, strin
             return Ok();
         }
 
-        // DELETE: api/ApiWithActions/5
+        // DELETE: api/ApiWithActions/5:w
+
         [HttpDelete("{id}")]
         public void Delete(int id)
         {
         }
+
+        [HttpGet]
+        [Route("{id}/followers")]
+        public IActionResult GetFollowers([FromRoute] Guid id)
+        {
+            return Ok(UserService.GetFollowers(id));
+        }
+
+        [HttpPut]
+        [Route("{id}/followers")]
+        public IActionResult AddFollower([FromRoute] Guid id, [FromBody] FollowerRequest follower)
+        {
+            // Guid FollowerId = Guid.Parse(followerId);
+            bool result = UserService.AddFollower(id, follower.Id);
+
+            if (!result)
+                return BadRequest();
+
+            return Ok();
+        }
+
+        [HttpDelete]
+        [Route("{id}/followers")]
+        public IActionResult RemoveFollower([FromQuery] Guid id, [FromBody] string followerId)
+        {
+            Guid FollowerId = Guid.Parse(followerId);
+            bool result = UserService.AddFollower(id, FollowerId);
```

Comment:
Acho que confundiu aqui
```suggestion
            bool result = UserService.RemoveFollower(id, FollowerId);
```

Reply:
Boa, ia passar despercebido!